app/models/tag.rb

```diff
@@ -23,6 +23,68 @@ class Tag < ActiveRecord::Base
   # the order is important, and it is the order in which they appear in the tag wrangling interface
   USER_DEFINED = ['Fandom', 'Character', 'Relationship', 'Freeform']
 
+  def is_this_a_large_tag(length_of_time_in_cache,how_long_it_takes_to_compute_size)
+    if length_of_time_in_cache < 40 || how_long_it_takes_to_compute_size < ( ArchiveConfig.TAGINGS_COUNT_DEFAULT_FOR_LARGE_TAGS || 4 )
```

Comment:
I don't think there should be a space in these parentheses, but also, what's the significance of `4` here? And I think it should be `TAGGINGS` with two Gs, not `TAGINGS` with one G.